_pages/community.html

```diff
@@ -35,6 +35,11 @@ <h5 class="mb-1">CompSoc Offers &raquo;</h5>
                         <h5 class="mb-1">School of Informatics &raquo;</h5>
                         <p class="mb-0">A group solely for the School of Informatics' students and staff.</p>
                     </a>
+                    <!-- Year Groups -->
+                    <a class="list-group-item list-group-action" href="{{ site.baseurl }}/infer-groups">
+                        <h5 class="mb-1">Specific Year Group Facebook pages &raquo;</h5>
+                        <p class="mb-0">Facebook groups for each year group.</p>
+                    </a>
```

Comment:
What do you think about setting the title to `Informatics Cohorts` and description to `For each year of Informatics students.`?

Reply:
Not sure about Cohorts, but if we keep 'Facebook groups' in the description, it'll probably be fine

Reply:
the reason for this is to match up with [this commit](https://github.com/compsoc-edinburgh/website2017/commit/851d0e5a2b2995bbb23107c3217aac4f330ff921) (yes i created the commit after my comment, but it's what i had in mind)

It being a facebook group is already implied by the column title